### ISSUE TITLE
Website/Domain down

### ISSUE THREAD

**Issue by kibichev** [benvp/toniefy#2]
Hi i think the domain has expired , do you renew it anytime soon? Thanks

**Comment by kibichev** [comment]
Mistake . ignore

**Comment by benvp** [comment]
Yeah. Sorry about that but I had some legal concerns about hosting the solution in the cloud. That's why it's not available anymore. You can - however - host it yourself if you like.
